test/metrics-luatest/helper.lua

```diff
@@ -0,0 +1,13 @@
+-- Assert running tests with built-in metrics.
+local rock_utils = require('third_party.metrics.test.rock_utils')
+rock_utils.assert_builtin('metrics')
```

Comment:
Will the tests fail if a recent metrics module (with built-in overriding support, after https://github.com/tarantool/metrics/pull/440) is installed into the system?

Reply:
Well, yeah. Let's workaround it.

Reply:
Added handle call and comment